src/js/claims-status/containers/AdditionalEvidencePage.jsx

```diff
@@ -114,7 +114,7 @@ class AdditionalEvidencePage extends React.Component {
                   onDirtyFields={this.props.setFieldsDirty}/>
               </div>
             </div>
-            <div className="small-12 usa-width-one-third medium-4 columns">
+            <div className="small-12 usa-width-one-third medium-4 columns help-sidebar">
```

Comment:
out of curiosity, what's going on here? Are we using 2 separate grid systems?

Reply:
I think so, yeah. `usa-width-one-third` is the grid from USWDS, but the problem with that is it's not responsive, so we're still still using `small-12 medium-4 columns` from Foundation.

As I understand it, the goal is to use `usa-width-*` where possible, then flexbox where it makes sense, and only using Foundation as a fallback when neither option is really great.